src/index.ts

```diff
@@ -6,7 +6,7 @@ import EventProcessor from "./event_processor";
 import { CameraDetails, CameraId, isMotionEndEvent } from "./types";
 import VideoDownloader from "./video_downloader";
 
-const { CAMERAS, DOWNLOAD_PATH, PREFER_SMART_MOTION, MQTT_HOST, UNIFI_HOST, UNIFI_USER, UNIFI_PASS } = process.env;
+const { CAMERAS, DOWNLOAD_PATH, PREFER_SMART_MOTION, MQTT_HOST, UNIFI_HOST, UNIFI_USER, UNIFI_PASS, MQTT_PATH } = process.env;
```

Comment:
Good idea, but should have a default value. Something like
```
const MQTT_PATH = process.env.MQTT_PATH ?? "unifi/protect-downloader/";
```

Reply:
agree, I will work on it some more this week